lang/clojure/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 
 name                clojure
 conflicts           clojure-devel
-epoch               20110929
-version             1.8.0
+epoch               20180727
```

Comment:
Please do not change the value from what it was before. Just keep it at `20110929`.